pkg/assetfetcher/http.go

```diff
@@ -95,6 +95,7 @@ func (c *NexposeAssetFetcher) FetchAssets(ctx context.Context, siteID string) (<
 		return nil, errChan
 	}
 
+	stater.Count("totalassets.site:"+siteID, float64(siteAssetResp.Page.TotalResources))
```

Comment:
Task: This is not the correct way to add a tag to a stat. The signature takes a variadic string argument at the end for any tags you wish to include.

The correct form would be something like `stater.Count("totalassets", float64(siteAssetResp.Page.TotalResources), fmt.Sprintf("site:%s", siteID))`

Reply:
oof ok, let me fix that!

Reply:
done